### PR TITLE
Bugfix/12.1.0 hashed identifiers

### PR DESCRIPTION
**Pull request by weakbit**
Short description
-----------------
The hashed identifier feature which did not properly work was fixed before. But the boostqueue did not work properly with hashed identifiers, this should solve that.

I advice to remove the regular "identifier" in favour of always hashed identifiers as i do not see disadvantages

### CODE REVIEW

File: Classes/Service/ClientService.php [hunk 1]
```diff
@@ -56,7 +56,8 @@ public function runSingleRequest(string $url): int
     /**
      * Get a cllable client.
      *
-     * @throws \Exception
+     * @param string $domain
```
Comment (lochmueller) [inline Classes/Service/ClientService.php:59]: Please do not PHPDoc comments, that are already part of the PHP type hinting, if there are no more details in the comment (same next line)

File: Classes/Domain/Repository/CacheRepository.php [hunk 2]
```diff
@@ -44,11 +44,14 @@ public function findExpiredIdentifiers(): array
 
     /**
      * Get all the cache identifiers.
+     *
+     * @param bool $hashedIdentifier
+     * @return array<string>
      */
-    public function findAllIdentifiers(): array
+    public function findAllIdentifiers(bool $hashedIdentifier): array
```
Comment (lochmueller) [inline Classes/Domain/Repository/CacheRepository.php:51]: Please add " = false" to keep the public method signature.

**Comment by lochmueller** [comment]
Hey @weakbit I add two notes in your code. Thank you for your contribution.

**Comment by weakbit** [comment]
I found more issues with hashed identifiers.
As i unfortunately have no more time, i'll continue the fixes in a breaking fork for now which always relays on hashed identifiers

**Pull request closed without merging**